### PR TITLE
qslim

### PR DESCRIPTION
**Pull request by yfzhou42**

### CODE REVIEW

File: app/EdgeOperations2d/ShortestEdgeCollapse.cpp [hunk 1]
```diff
@@ -54,6 +54,7 @@ std::vector<TriMesh::Tuple> Edge2d::EdgeOperations2d::new_edges_after(
 
 bool Edge2d::EdgeOperations2d::collapse_shortest(int target_operation_count)
 {
+    size_t initial_size = get_vertices().size();
```
Comment (jiangzhongshi) [inline app/EdgeOperations2d/ShortestEdgeCollapse.cpp:57]: Why is this here? This result is not used in your code.

File: app/QSLIM/app/main.cpp [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+}
+int main(int argc, char** argv)
+{
+    // input
+    // target_verts
+    // output
+    // ep
+
+    const std::string root(WMT_DATA_DIR);
+    const std::string path = argv[1];
```
Comment (jiangzhongshi) [inline app/QSLIM/app/main.cpp:49]: prefer use cli11 instead of these confusing argv[x].

File: app/EdgeOperations2d/ShortestEdgeCollapse.cpp (hunk 1 above)
Comment (yfzhou42) [reply app/EdgeOperations2d/ShortestEdgeCollapse.cpp:57]: This is the old code. The whole /EdgeOPerations2d folder can be deleted soon

File: src/wmtk/ExecutionScheduler.hpp [hunk 3]
```diff
@@ -271,6 +272,10 @@ struct ExecutePass
 
                 if (stop.load(std::memory_order_acquire)) return;
                 if (cnt_update > stopping_criterion_checking_frequency) {
+                    wmtk::logger().info(
+                        "I am in stopping criteria cnt : {} freqency is :{}",
+                        cnt_success,
+                        stopping_criterion_checking_frequency);
```
Comment (jiangzhongshi) [inline src/wmtk/ExecutionScheduler.hpp:278]:
```suggestion
```
